src/navigationstacks/AuthenticationStack.tsx

```diff
@@ -11,7 +15,24 @@ export const AuthenticationStack = ({ stack }) => {
   return (
     <ViewLayoutStructure navbarVisible={false}>
       <stack.Navigator
-        screenOptions={{ headerShown: true, animation: "fade" }}
+        screenOptions={{
+          headerShown: true,
+          animation: "fade",
+          header: ({ navigation, route, options, back }) => {
+            const title = getHeaderTitle(options, route.name);
+
+            return (
+              <TopNavBar
+                leftSection={
+                  back ? <BackButton navigation={navigation} /> : undefined
+                }
+                rightSection={<TopNavBarRightLogin />}
+                title={title}
+                style={undefined}
```

Comment:
po co nam style w komponencie jak nie jest korzystany?

Reply:
Racja, poprawie to